libpod/info.go

```diff
@@ -29,6 +30,11 @@ func (r *Runtime) hostInfo() (map[string]interface{}, error) {
 	info["arch"] = runtime.GOARCH
 	info["cpus"] = runtime.NumCPU()
 	info["rootless"] = rootless.IsRootless()
+	unified, err := cgroups.IsCgroup2UnifiedMode()
+	if err != nil {
+		return nil, errors.Wrapf(err, "error reading cgroups mode")
+	}
+	info["cgroupv2"] = unified
```

Comment:
cgroup2? cgroupv2?

Reply:
Maybe we should set this to a string instead - "cgroupmode" = "v2" or "v1"?

Reply:
legacy/hybrid/unified?

Reply:
i like the idea from @mheon

Reply:
the issue is that currently we have only code to detect the unified mode (that is cgroups v2).  Should we probably rename it to "cgroups-unified-mode"?

Reply:
Users understand cgroups V1 and V2,  I don't want the "unified" mode exposed.  When I see unified, I get confused and think it is a mode that covers both V1 and V2.  

"CgroupVersion"  = V2

That way 5 years from now when V3 is all the rage ...

Reply:
fixed to be `"CgroupVersion" = "v1" | "v2"`